src/org/pentaho/reporting/platform/plugin/async/PentahoAsyncReportExecution.java

```diff
@@ -45,10 +45,6 @@ public InputStream call() throws Exception {
 
       if ( reportComponent.execute() ) {
 
-        final long end = System.currentTimeMillis();
-        AuditHelper.audit( userSession, userSession, url, getClass().getName(), getClass()
-          .getName(), MessageTypes.INSTANCE_END, instanceId, "", ( (float) ( end - start ) / 1000 ), null );
-
         listener.setStatus( AsyncExecutionStatus.FINISHED );
```

Comment:
What is the reason that we we no longer log the Audit information for the success of the execution?